packages/kotti-ui/source/kotti-field-inline-edit/components/EditIcon.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+	<span :class="classes" role="button" :tabindex="-1" v-text="Yoco.Icon.EDIT" />
```

Comment:
could be
```suggestion
	<span class="kt-field-inline-edit__edit-icon yoco" role="button" :tabindex="-1" v-text="Yoco.Icon.EDIT" />
```
to remove the computed